arduino/obstacleAvoidance/basicObstacleAvoidance.ino

```diff
@@ -0,0 +1,47 @@
+#include <Smartcar.h>
+#include <Wire.h>
+#include <VL53L0X.h>
+
+const int START_SPEED = 50;
+const int MIN_OBSTACLE_DISTANCE = 300;
+const int BAUD_RATE = 9600;
+const int STOP = 0;
+
+BrushedMotor leftMotor(smartcarlib::pins::v2::leftMotorPins);
+BrushedMotor rightMotor(smartcarlib::pins::v2::rightMotorPins);
+DifferentialControl control(leftMotor, rightMotor);
+
+SimpleCar car(control);

```

Comment:
kudos for using the `SimpleCar` since you do not need any more functionality (e.g. orientation) at the moment. :+1: